.gitlab/jobs/lassen.yml

```
@@ -58,5 +58,5 @@ clang_13_0_1_libcpp:
 
 gcc_8_3_1_cuda_10_1_243_desul_atomics:
   variables:
-    SPEC: " ~shared +openmp +tests +cuda +desul %gcc@8.3.1 cuda_arch=70 ^cuda@10.1.243+allow-unsupported-compilers ^blt@git.a7f0a6ecc4fdfa1724399b1454c3909b9ee02e81=develop"
+    SPEC: " ~shared +openmp +tests +cuda +desul %gcc@8.3.1 cuda_arch=70 ^cuda@10.1.243+allow-unsupported-compilers ^blt@git.6dc747989504adee075bbe2ebf703a230232bea5=develop"
```

BLT is updated, but it's pulling the old desul now.  
```suggestion
    SPEC: " ~shared +openmp +tests +cuda +desul %gcc@8.3.1 cuda_arch=70 ^cuda@10.1.243+allow-unsupported-compilers ^blt@git.6dc747989504adee075bbe2ebf703a230232bea5=develop ^desul@git.a0b1b2fadded49ef526e31aa296218f565a6f5d1=develop"
```

---

No, this is not what's happening because desul isn't split into a spack package in the package version we have... how is this failing...

---

I do not know. Test output indicates CUDA kernels with atomics aren't doing anything.  In a local config on rzansel, ccmake tells me that `DESUL_ENABLE_OPENMP` is ON. That's not what we want, correct? Is something off by trying to shut that off?

I'm trying something else locally on rzansel now.....

---

@trws when I set `ENABLE_OPENMP=Off` in my CMake recipe, all tests pass. Also, all the compiler warnings coming from the CUDA atomic tests about calling a host function from a host-device function go away. 

The issue seems to be that building with both CUDA and OpenMP with desul atomics enabled breaks the CUDA atomics.